clarify and simplify trip extraction rules (#2)
vimcar/backend-challenge-trip-extraction-2021: readme.md

# Vimcar Coding Challenge: Trip Extraction

One of Vimcar's successful products is the digital logbook. The purpose of a logbook is to maintain a complete, consistent list of all trips a vehicle has made over time.

There are many different use-cases for this, one of them is taxation. Our customers e.g. maintain a logbook, to keep track of how many trips were made privately and how many kilometers were driven in a business context.

## The trip extraction

> The central piece of the logbook is the trip extraction. Trip extraction is the process of determining when a vehicle starts and stops moving based on data collected by gps trackers (Waypoints).

Reviewer: ```suggestion
The central piece of the logbook is trip extraction. Trip extraction is the process of determining when a vehicle starts and stops moving based on data collected by GPS trackers. The data from the GPS is a collection of waypoints collected at semi-regular intervals.
```

> When driving from A to B, a user will probably have interruptions like stopping at red lights, traffic jams, tunnels. The expected behavior is to have one trip accounting for the whole journey, and not broken trips like from A to first red light etc.

Reviewer: ```suggestion
When driving from A to B, a user will probably have interruptions to their movement like stopping at red lights and traffic jams. The expected behavior is to have one trip accounting for the whole journey, and not broken trips like from A to first red light etc.
```

> GPS technology has some limitations which the trip extraction should account for. Within cities the GPS precision may be poorer than 15 meters and the GPS position might even "jump", due to bad reception caused by reflections on building. There are places without gps signal like tunnels and underground parking lots.

Reviewer: ```suggestion
GPS technology has some limitations which the trip extraction should account for. Within cities the GPS precision may be poorer than 15 meters and the GPS position might even "jump", due to bad reception caused by reflections on building. There are places without GPS signal like tunnels and underground parking lots.
```
